code/modules/randomMaps/vault_definitions.dm

```diff
@@ -13,11 +13,17 @@ var/list/existing_vaults = list()
 
 /datum/vault/proc/initialize(list/objects)
 	existing_vaults.Add(src)
-
+	
 	for(var/turf/new_turf in objects)
-		if(!src.location) location = new_turf
-
+		if(!location) location = new_turf
 		new_turf.flags |= NO_MINIMAP //Makes the spawned turfs invisible on minimaps
```

Comment:
Put this after the if block for base turf stuff so that flags aren't overwritten by ChangeTurf if you want to prevent them from being on the minimap as well